backend/packages/Upgrade/src/init/seed/initMetrics.ts

```diff
@@ -8,7 +8,9 @@ export function InitMetrics(logger: UpgradeLogger): Promise<any> {
   // Init default metrics in system
   if (env.initialization.metrics) {
     try {
-      return metricService.saveAllMetrics(JSON.parse(env.initialization.metrics), logger);
+      return JSON.parse(env.initialization.metrics).map((metricData) => {
```

Comment:
This is not returning a Promise but an array of Promise. You should use `Promise.all` before returning

Reply:
code updated